src/components/AcceptDialog.tsx

```diff
@@ -12,13 +12,38 @@ import {
 import { DocumentState } from '../domain/document-submission';
 import DateFields from './DateFields';
 import router from 'next/router';
+import { DateTime } from 'luxon';
 
 const schema = Yup.object().shape({
   staffSelectedDocumentTypeId: Yup.string().required(
     'Please choose a document type'
   ),
+  validUntilDates: Yup.array().test(
```

Comment:
The reason why I've had to write a .test instead of .date is because the default behaviour of .date in Yup is to parse it to a date, but as mm-dd-yyyy, not dd-mm-yyyy. Also, it didn't seem to capture 'impossible dates' like 32nd Feb.